vkubelet/apiserver.go

```diff
@@ -0,0 +1,43 @@
+package vkubelet
+
+import (
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+)
+var p Provider
+
+func ApiserverStart(provider Provider) error {
+	p = provider
+	http.HandleFunc("/", ApiServerHandler)
+	certFilePath := os.Getenv("APISERVER_CERT_LOCATION")
+	keyFilePath := os.Getenv("APISERVER_KEY_LOCATION")
+	err := http.ListenAndServeTLS(":10250", certFilePath, keyFilePath, nil)
+	if err != nil {
+		log.Fatal(err)
+	}
+	return err
+}
+
+func ApiServerHandler(w http.ResponseWriter, req *http.Request) {
+	if req.Method == "GET" {
+		if strings.ContainsAny(req.RequestURI, "containerLogs" ) {
```

Comment:
not familiar with go, but this looks quite raw. @erikstmartin, is there any go package which helps to parse the URI and route to different webapi?

Reply:
https://golang.org/pkg/net/http/#Handle allows you to give it the full path you're looking for. We likely want to use this because there will be other endpoints and we can have different functions handle each route.

Also this current implementation would match a url that has containerLogs anywhere in it